company_website/static/common/scripts/gtag_setup.js

```diff
@@ -0,0 +1,6 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){
+  dataLayer.push(arguments);
+}
+gtag('js', new Date());
+gtag('config', 'AW-607460288');
```

Comment:
Same here about `AW-607460288` parameter. It should be in my opinion be injected here instead of hardcoded here